init.sh

```diff
@@ -4,6 +4,12 @@ BRANCH=${REPO_BRANCH:='master'}
 INV=${INVENTORY:="/root/inventory.yaml"}
 VALS=${VALIDATIONS:=""}
 
+# Set logging env var to file mounted into the container.
+if [ -f "/root/validations.log" ]; then
+  export ANSIBLE_LOG_PATH=${LOG_PATH}
```

Comment:
this will fail - you want to put the value, not an unset var ;).

Reply:
d'oh